### PR TITLE
Apply (mostly)trivial clang-tidy in llmq folder

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/llmq/quorums_blockprocessor.cpp [hunk 1]
```diff
@@ -379,7 +379,7 @@ uint256 CQuorumBlockProcessor::GetQuorumBlockHash(Consensus::LLMQType llmqType,
     int quorumStartHeight = nHeight - (nHeight % params.dkgInterval);
     uint256 quorumBlockHash;
     if (!GetBlockHash(quorumBlockHash, quorumStartHeight)) {
-        return uint256();
+        return {};
```
Comment (xdustinface) [inline src/llmq/quorums_blockprocessor.cpp:382]: Whats your reasoning for all the `uint256() -> {}`?

File: src/llmq/quorums_signing.cpp [hunk 2]
```diff
@@ -575,7 +574,7 @@ void CSigningManager::CollectPendingRecoveredSigsToVerify(
         for (auto it = v.begin(); it != v.end();) {
             auto& recSig = *it;
 
-            Consensus::LLMQType llmqType = (Consensus::LLMQType) recSig.llmqType;
+            auto llmqType = (Consensus::LLMQType) recSig.llmqType;
```
Comment (xdustinface) [inline src/llmq/quorums_signing.cpp:577]: why not just
```suggestion
            Consensus::LLMQType llmqType = recSig.llmqType;
```
or
```suggestion
            auto llmqType = recSig.llmqType;
```

I anyway asked myself many times already why we have so many more places in the code where we cast variables from type `Consensus::LLMQType` to `Consensus::LLMQType` 😄 Doesn't make a lot sense imo unless thats some fancy practice im not aware of.

File: src/llmq/quorums_blockprocessor.cpp (hunk 1 above)
Comment (xdustinface) [reply src/llmq/quorums_blockprocessor.cpp:382]: Wait thats a dump question probably as this it a clang-tidy cleanup, but still.. really? Thats something we should follow? 😆 I don't like it haha.. but.. OK.

File: src/llmq/quorums_signing.cpp (hunk 2 above)
Comment (PastaPastaPasta) [reply src/llmq/quorums_signing.cpp:577]: Hmm, I'm not sure why we do that... Just know this resolves the clang warning

**Pull request merged**